src/booleans.js

```diff
@@ -1,62 +1,94 @@
 function negate(a) {
   // your code here
+  return !(a > 0);
```

Comment:
While this works, it isnt the clearest of logic as the value being passed in is a boolean and not a number. Instead you can just return the inverse of the input like this:
```
  return !a;
```